src/WebClientPool.ts

```diff
@@ -0,0 +1,15 @@
+import { WebClient, WebClientOptions } from '@slack/web-api';
```

Comment:
I like this idea but how about doing this in another (smaller) pull request?

Reply:
its related to getting the lint rules to pass. the rule is [`max-classes-per-file`](https://eslint.org/docs/rules/max-classes-per-file). we could do this in a separate PR if we want to land it before this PR. otherwise this PR will be blocked on failing its own lint.

Reply:
Thanks for sharing the context. 

> we could do this in a separate PR if we want to land it before this PR

I think there is no blocker for it. If we separate this change, we can quickly review and merge it. Also, if the change has issues by any chance, it's much easier to revert.